src/lua/init.c

```diff
@@ -926,6 +946,12 @@ tarantool_lua_init(const char *tarantool_bin, int argc, char **argv)
 	tarantool_L = L;
 }
 
+void
+tarantool_lua_post_box_init(lua_State *L)
+{
+	tarantool_lua_load_modules(L, lua_modules_post_box);
+}
```

Comment:
We can just add it into src/box/lua/init.c after 5th commit from #8250. Let's wait a bit or rebase on top of the pull request.

Reply:
Reworked the PR based on #8250 , please review (I don't see a way to set the merge into your fork branch, so PR have 8 commits now). [2]